### DISCUSSION TITLE
General Chat

### DISCUSSION THREAD

**Discussion by ryanhtafesa** [ryanhtafesa/T24S2T1EXTG04-REV2#3] [General]
Hi @AlexandraKoshur 

Now that we are working through this again, I have come across a few things to watch out for. Subtle things which can easily trip things up. Can you watch out for?

- Update your connection settings to the new github URL link
- I found it helpful so far to triple check copy the revision numbers in order into a notepad. This way I can cross reference each step 
- Be super careful with the cloning step. It can be misleading when it only downloads the main branch by default. 

Let me know if you get stuck. 
Ryan

**Comment by AlexandraKoshur** [comment]
@ryanhtafesa, thanks for the heads-up! Cloned successfully

**Comment by AlexandraKoshur** [comment]
@ryanhtafesa,
I keep having problems with the code after merging feature_currency to development branch. Trying to figure out what is wrong so we could work on the project further.

**Reply by ryanhtafesa** [reply]
@AlexandraKoshur 
Looks like we were trying to solve the conflicts incorrect. 
Now that we better understand the colouring system of the merge conflicts better, We just need to insert your currency features into the development code that exists. 

As per Teams, it sounds like we both have a working local repo now with the changes. 
So hopefully we can now merge to release ! Thanks for taking the time!

**Comment by AlexandraKoshur** [comment]
@ryanhtafesa, I pushed the development branch. Please, let me know when you 'll get a chance to merge it further and we can start section G!

**Reply by ryanhtafesa** [reply]
Thanks @AlexandraKoshur 
All merged to the final stage now, and tagged V2.0
Have pushed all back to GitHub ready for the next steps
